examples/client/client.c

```diff
@@ -329,6 +329,29 @@ static void SetKeyShare(WOLFSSL* ssl, int onlyKeyShare, int useX25519,
 #endif
 
 #ifdef WOLFSSL_EARLY_DATA
+static void EarlyDataStatus(WOLFSSL* ssl)
```

Comment:
Suggest moving this to test.h where other common printing functions are.

Reply:
Thank you for pointing it out. Moved to test.h